src/router.rs

```diff
@@ -9,6 +9,8 @@ use crate::agent::{Agent, AgentMsg, AnyMessage};
 use crate::errors::{Error, Result};
 use crate::server::ConnectionAddr;
 use tokio::spawn;
+use flume::Receiver;
+use flume::Sender;
```

Comment:
Keeping it consistent
```suggestion
use flume::{Receiver, Sender};
```